### PR TITLE
Updates README.

### PR DESCRIPTION
**Pull request by klizhentas**
Updates hero image, removes happy talk, uses new language focusing on protection.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,26 @@
...
+
+Here is why you might use Teleport:
+
+* Set up SSO for all your cloud infrastructure [1].
+* Protect access for engineers across all services, both in the cloud and on-prem, using mTLS endpoints and short-lived certificates.
+* Establish tunnels to access services behind NATs and firewalls.
+* Provide an audit log with session recording and replay for various protocols.
+* Role-based access control (RBAC) and [access requests](https://goteleport.com/features/access-requests/).
+
+[1] Open Source version supports only Github SSO.
```
Comment (strideynet) [inline README.md:13]:
```suggestion
[1] Open Source version supports only GitHub SSO.
```

File: README.md [hunk 2]
```diff
@@ -37,42 +53,32 @@
...
-access to behind-NAT resources such as:
+You can read more about its architecture [here](https://goteleport.com/docs/architecture/).
 
-* [SSH nodes](https://goteleport.com/docs/getting-started/) - SSH works in browsers too!
-* [Kubernetes clusters](https://goteleport.com/docs/kubernetes-access/introduction/)
-* [PostgreSQL, MongoDB, CockroachDB and MySQL databases](https://goteleport.com/docs/database-access/introduction/)
-* [Internal Web apps](https://goteleport.com/docs/application-access/introduction/)
-* [Windows Hosts](https://goteleport.com/docs/desktop-access/introduction/)
-* [Networked servers](https://goteleport.com/docs/server-access/introduction/)
+We have implemented Teleport as a single Golang binary that integrates with multiple protocols and cloud services:
```
Comment (strideynet) [inline README.md:60]:
```suggestion
We have implemented Teleport as a single Go binary that integrates with multiple protocols and cloud services:
```

File: README.md [hunk 3]
```diff
@@ -37,42 +53,32 @@
...
-- Single sign-on (SSO) for everything via GitHub Auth, OpenID Connect, or SAML with endpoints like Okta or Active Directory.
+- Single sign-on (SSO) for everything via GitHub Auth, OpenID Connect, or SAML with endpoints like Okta or Active Directory [2]
 - Infrastructure introspection: Use Teleport via the CLI or Web UI to view the status of every SSH node, database instance, Kubernetes cluster, or internal web app.
 
-Teleport is built upon the high-quality [Golang SSH](https://godoc.org/golang.org/x/crypto/ssh)
-implementation. It is _fully compatible with OpenSSH_,
-`sshd` servers, and `ssh` clients.
+[2] Open-Source version only supports Github SSO.
+
+Teleport uses [Golang crypto](https://godoc.org/golang.org/x/crypto). It is _fully compatible with OpenSSH_, `sshd` servers, and `ssh` clients, Kubernetes clusters and more.
```
Comment (strideynet) [inline README.md:81]:
```suggestion
Teleport uses [Go crypto](https://godoc.org/golang.org/x/crypto). It is _fully compatible with OpenSSH_, `sshd` servers, and `ssh` clients, Kubernetes clusters and more.
```

File: README.md [hunk 4]
```diff
@@ -1,6 +1,26 @@
+Teleport protects access to your infrastructure.
```
Comment (strideynet) [inline README.md:1]: Nit: It feels a little odd to have this prose above the hero image/repository badges

**Review by strideynet — APPROVED** [review]

File: README.md (hunk 4 above)
Comment (zmb3) [reply README.md:1]: +1

File: README.md [hunk 5]
```diff
@@ -1,6 +1,26 @@
...
+
+It provides connectivity, authentication, access controls and audit.
+
+Here is why you might use Teleport:
+
+* Set up SSO for all your cloud infrastructure [1].
```
Comment (zmb3) [inline README.md:7]:
```suggestion
* Set up SSO for all of your cloud infrastructure [1].
```

File: README.md [hunk 6]
```diff
@@ -1,6 +1,26 @@
...
+* Protect access for engineers across all services, both in the cloud and on-prem, using mTLS endpoints and short-lived certificates.
```
Comment (zmb3) [inline README.md:8]:
```suggestion
* Protect access to all services (both in the cloud and on-prem) using mTLS endpoints and short-lived certificates.
```

File: README.md [hunk 7]
```diff
@@ -1,6 +1,26 @@
...
+* Establish tunnels to access services behind NATs and firewalls.
+* Provide an audit log with session recording and replay for various protocols.
+* Role-based access control (RBAC) and [access requests](https://goteleport.com/features/access-requests/).
```
Comment (zmb3) [inline README.md:11]: I think this one is missing the _why_.

File: README.md [hunk 8]
```diff
@@ -1,6 +1,26 @@
...
+
+[1] Open Source version supports only Github SSO.
+
+Teleport works with SSH, Kubernetes, databases, RDP, and web services, integrates
+with cloud and self-hosted infra.
```
Comment (zmb3) [inline README.md:16]:
```suggestion
Teleport works with SSH, Kubernetes, databases, RDP, and web services.
```

We already mentioned cloud and self-hosted infra (8 lines above).

File: README.md [hunk 9]
```diff
@@ -37,42 +53,32 @@
 
 ## Introduction
 
-Teleport is the easiest, most secure way to access all your infrastructure.
-Teleport is an identity-aware, multi-protocol access proxy which understands
-SSH, HTTPS, RDP, Kubernetes API, MySQL, MongoDB and PostgreSQL wire protocols.
+Teleport includes an identity-aware access proxy, a CA that issues short-lived certificates, and an access control and tunneling system to access resources behind the firewall.
```
Comment (zmb3) [inline README.md:56]: The access control system and tunneling system are separate things. This makes them sound the same. (You can use access control without tunneling)

File: README.md [hunk 10]
```diff
@@ -37,42 +53,32 @@
...
 
-On the server-side, Teleport is a single binary which enables convenient secure
-access to behind-NAT resources such as:
+You can read more about its architecture [here](https://goteleport.com/docs/architecture/).
```
Comment (zmb3) [inline README.md:58]: We already linked to this on line 18.

File: README.md [hunk 11]
```diff
@@ -37,42 +53,32 @@
...
-* [Kubernetes clusters](https://goteleport.com/docs/kubernetes-access/introduction/)
-* [PostgreSQL, MongoDB, CockroachDB and MySQL databases](https://goteleport.com/docs/database-access/introduction/)
-* [Internal Web apps](https://goteleport.com/docs/application-access/introduction/)
-* [Windows Hosts](https://goteleport.com/docs/desktop-access/introduction/)
-* [Networked servers](https://goteleport.com/docs/server-access/introduction/)
+We have implemented Teleport as a single Golang binary that integrates with multiple protocols and cloud services:
 
-Teleport is trivial to set up as a Linux daemon or in a Kubernetes pod. It's rapidly
-replacing legacy `sshd`-based setups at organizations who need:
+* [SSH nodes](https://goteleport.com/docs/getting-started/).
```
Comment (zmb3) [inline README.md:62]: Not sure why we're linking to this page for "SSH nodes" - it hardly mentions SSH.

Also, we've already included this link on line 19 so we probably don't need to include it twice in such a short document.

File: README.md [hunk 12]
```diff
@@ -37,42 +53,32 @@
...
-  across many environments and cloud providers.
-* Audit log with session recording/replay for multiple protocols
-* Easily manage trust between teams, organizations and data centers.
-* Role-based access control (RBAC) and flexible access workflows (one-time [access requests](https://goteleport.com/features/access-requests/))
+You can set up Teleport as a [Linux daemon](https://goteleport.com/docs/#set-up-a-demo-cluster) or a [Kubernetes deployment](https://goteleport.com/docs/deploy-a-cluster/helm-deployments/).
 
-In addition to its hallmark features, Teleport is interesting for smaller teams
-because it facilitates easy adoption of the best infrastructure security
-practices like:
+Teleport focuses on the best infrastructure security practices:
```
Comment (zmb3) [inline README.md:71]:
```suggestion
Teleport focuses on best practices for infrastructure security:
```

File: README.md [hunk 13]
```diff
@@ -37,42 +53,32 @@
...
 
-- No need to manage shared secrets such as SSH keys: Teleport uses certificate-based access with automatic certificate expiration time for all protocols.
+- No need to manage shared secrets such as SSH keys or Kubernetes tokens: it uses certificate-based auth with certificate expiration for all protocols.
 - Two-factor authentication (2FA) for everything.
 - Collaboratively troubleshoot issues through session sharing.
-- Single sign-on (SSO) for everything via GitHub Auth, OpenID Connect, or SAML with endpoints like Okta or Active Directory.
+- Single sign-on (SSO) for everything via GitHub Auth, OpenID Connect, or SAML with endpoints like Okta or Active Directory [2]
```
Comment (zmb3) [inline README.md:76]: I wouldn't consider Okta and Active directory "endpoints"

File: README.md [hunk 14]
```diff
@@ -37,42 +53,32 @@
...
 - Infrastructure introspection: Use Teleport via the CLI or Web UI to view the status of every SSH node, database instance, Kubernetes cluster, or internal web app.
 
-Teleport is built upon the high-quality [Golang SSH](https://godoc.org/golang.org/x/crypto/ssh)
-implementation. It is _fully compatible with OpenSSH_,
-`sshd` servers, and `ssh` clients.
+[2] Open-Source version only supports Github SSO.
```
Comment (zmb3) [inline README.md:79]: This was already mentioned on line 13.

File: README.md (hunk 3 above)
Comment (zmb3) [reply README.md:81]: I would also say "it is compatible with OpenSSH" - not "fully compatible".

"fully" is just a filler word in this case - the sentence reads just the same without it, and then you don't open up debate as to what "fully" actually means.

**Review by zmb3 — APPROVED** [review]

**Review by strideynet — APPROVED** [review]
👍  

My only thought is whether we should move the introductory text below the hero image/badges - especially as the hero image essentially serves as a header.

**Comment by klizhentas** [comment]
@strideynet I'd err on the side of giving people easy way to read about Teleport first

**Pull request merged**